docs/platform-external/installing.md

```diff
@@ -0,0 +1,455 @@
+      serviceAccount: cloud-controller-manager
+      priorityClassName: system-cluster-critical
+      nodeSelector:
+        node-role.kubernetes.io/master: ""
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - topologyKey: "kubernetes.io/hostname"
+            labelSelector:
+              matchLabels:
+                k8s-app: {{ cloud provider name }}-cloud-controller-manager # replace me
+                infrastructure.openshift.io/cloud-controller-manager: {{ cloud provider name }} # replace me
+      tolerations:
+        - key: CriticalAddonsOnly
+          operator: Exists
+        - key: node-role.kubernetes.io/master
+          operator: Exists
+          effect: "NoSchedule"
+        - key: node.cloudprovider.kubernetes.io/uninitialized
+          operator: Exists
+          effect: NoSchedule
+        - key: node.kubernetes.io/not-ready
+          operator: Exists
+          effect: NoSchedule
+      containers:
+        - name: cloud-controller-manager
+          image: {{ cloud controller image URI }} # replace me
+          imagePullPolicy: "IfNotPresent"
+          command:
+          - /bin/bash
+          - -c
+          - |
+            #!/bin/bash
+            set -o allexport
+            if [[ -f /etc/kubernetes/apiserver-url.env ]]; then
+              source /etc/kubernetes/apiserver-url.env
+            fi
+            exec {{ cloud controller binary }} # replace me
+          ports:
+          - containerPort: 10258
+            name: https
+            protocol: TCP
+          resources:
+            requests:
+              cpu: 200m
+              memory: 50Mi
+          volumeMounts:
+            - mountPath: /etc/kubernetes
+              name: host-etc-kube
+              readOnly: true
+            - name: trusted-ca
+              mountPath: /etc/pki/ca-trust/extracted/pem
+              readOnly: true
+            # mount extra volumes if needed
+      volumes:
+        - name: host-etc-kube
+          hostPath:
+            path: /etc/kubernetes
+            type: Directory
+        - name: trusted-ca
+          configMap:
+            name: ccm-trusted-ca
+            items:
+              - key: ca-bundle.crt
+                path: tls-ca-bundle.pem
+        # add extra volumes if needed
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: system:cloud-controller-manager
+  labels:
+    kubernetes.io/cluster-service: "true"
+rules:
+  {{ define here the authorization rules required by your CCM }}
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: oci-cloud-controller-manager
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: system:cloud-controller-manager
+subjects:
+- kind: ServiceAccount
+  name: cloud-controller-manager
+  namespace: {{ cloud provider name }}-cloud-controller-manager
+```
+
+
+#### Create custom manifests for Kubelet
```

Comment:
will we need these instructions with the external changes we are landing in 4.14?

Reply:
Yes, these are instructions to provide the ProviderID through MachineConfig, I think it is not part of the code changes but for the installation setup. Am I correct?